test/classes/claridge/baseConstructorCall.chpl

```diff
@@ -1,17 +1,18 @@
 class BaseClass {
   var a, b: int;
 
-  proc BaseClass(b: int) {
+  proc init(b: int) {
     this.b = b;
+    super.init();
```

Comment:
Not necessary, but I think it is preferable, since it means the statement above it executes in Phase 1 instead of having b get an initial value that will just get overwritten